src/Controllers/CopyPatrol.php

```diff
@@ -87,7 +87,34 @@ public function setWikiprojectDao( $wikiprojectDao ) {
 	 * editor_talk_dead: Bool. Is editor talk page non-existent?
 	 */
 	protected function handleGet() {
-		$records = $this->dao->getPlagiarismRecords();
+		$userData = $this->authManager->getUserData();
+		$currentUser = $userData ? $userData->getName() : NULL;
+		$filter = 'all';
+		$filterUser = NULL;
+
+		// set filter types and descriptions that will be rendered as checkboxes in the view
+		$filterTypes = array(
+			'all' => 'All cases',
+			'open' => 'Open cases',
+			'fixed' => 'All "page fixed" cases',
+			'noaction' => 'All "no action needed" cases'
+		);
+
+		// use given filter if set, or 'all' if filter is 'mine' and they are not logged in
+		if ( isset( $_GET['filter'] ) ) {
+			if ( $_GET['filter'] === 'mine' ) {
+				if ( isset( $currentUser ) ) {
+					$filter = 'mine';
+					$filterUser = $currentUser;
+					$filterTypes['mine'] = 'My reviews'; // add 'My reviews' to filter options
```

Comment:
It looks like they only get the 'My reviews' option if they've already set the filter to 'mine', which doesn't make sense.